dev/ci/cases/pr/C96mx025_S2S.yaml

```
@@ -0,0 +1,29 @@
+experiment:
+  net: sfs
+  mode: forecast-only
+  idate: 1994050100
+  edate: 1994050100
+  pslot: {{ 'pslot' | getenv }}
+  app: S2S
+  resdetatmos: 96
+  resensatmos: 96
+  resdetocean: 0.25 
+  start: 'cold'
+  nens: 2
+  comroot: {{ 'RUNTESTS' | getenv }}/COMROOT
+  expdir: {{ 'RUNTESTS' | getenv }}/EXPDIR
+  icsdir: {{ 'TOPICDIR' | getenv }}/CPC/C96mx025
+#  icsdir: {{ 'ICSDIR_ROOT' | getenv }}/C96mx025/20250808
```

@NeilBarton-NOAA I have staged the data on all platforms. Please update this to
```suggestion
  icsdir: {{ 'ICSDIR_ROOT' | getenv }}/C96mx025/20251217
```